src/updateAttribute.js

```diff
@@ -49,6 +49,29 @@ function setAttribute (node, attrName, attrValue, oldAttrValue) {
     } else {
```

Comment:
Check how styles are handled on preact to cover all the cases with style attribute.
https://github.com/preactjs/preact/blob/master/src/diff/props.js